website/server/controllers/api-v3/tasks.js

```diff
@@ -5,6 +5,7 @@ import {
 } from '../../libs/webhook';
 import { removeFromArray } from '../../libs/collectionManipulators';
 import * as Tasks from '../../models/task';
+import * as groupTasks from '../../libs/groupTasks';
```

Comment:
Change these to only import what you need. So this should be `import { handleSharedCompletion } from  '../../libs/groupTasks';`